addon/utils/url-builder.js

```diff
@@ -1,8 +1,22 @@
-import param from 'jquery-param';
+import URL from 'url';
 
 export default function(url, path, queryParams) {
-  let query = param(queryParams);
-  let pathUrl = url.charAt(url.length - 1) === '/' ? `${url}${path}` : `${url}/${path}`;
+  let pathUrl;
+  if (path) {
+    let maybeMissingSlash = url.endsWith('/') || path.startsWith('/') ? '' : '/'
+    pathUrl = `${url}${maybeMissingSlash}${path}`;
+  } else {
+    pathUrl = url
+  }
+
+  // Use native URL API to generate query
+  let queryObj = new URL('https://exelord.com/ember-custom-actions/').searchParams;
```

Comment:
```suggestion
  // Passing in example.com because a domain is needed, but we're not using it.
  // Using `URL` only to access the queryParams API.
  let queryObj = new URL('https://example.com').searchParams;
```

Reply:
`https://example.test` - never use `.com` it fires DNS lookup, which allows track where the request is coming from.

Reply:
:+1:

Reply:
In case we won't render it that's true. But just to prevent that issue in the future is better to forget about .com's for examples :)